10_transformers-from-scratch.ipynb

```diff
@@ -193,6 +193,13 @@
     "#### Memory mapping"
```

Comment:
<p>I suggest tweaking the text as follows:</p><p> </p><blockquote><strong>Note:</strong> The following code block assumes that you have downloaded the BigQuery dataset to a folder called <code>codeparrot</code>. We suggest skipping this step since it will unpack the compressed files and require ~180GB of disk space.  This code is just for demonstration purposes and you can just continue below with the streamed dataset which will not consume that much disk space.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>